app/views/appointments/edit.html.erb

```diff
@@ -3,6 +3,18 @@
   { title: "Edit appointment for #{@appointment.name}" }
 ) %>
 
+<% if @appointment.imported? %>
+  <div class="alert alert-warning t-appointment-was-imported-message" role="alert">
+    <p>
+      This appointment was automatically imported from Booking Bug,
+      which means that the date of birth is not valid.
+    </p>
+    <p>
+      Do not read the date of birth out to the customer.
```

Comment:
Such firmness 😤😡😡 

Maybe "Please refer to the date of birth in the notes field" or something?